and-cli-install.js

```diff
@@ -48,8 +48,8 @@ require("./command-runner").run(async () => {
 
             // Developer alias
             echo.message("Configuring cli development `and-cli-dev` bash alias...");
-            const pathToCli      = upath.toUnix(path.join(shell.pwd().toString(), "cli.js"));
-            const developerAlias = `alias and-cli-dev = '${pathToCli}'`;
+            const pathToCli      = upath.toUnix(path.join(shell.pwd().toString(), "and-cli.js"));
+            const developerAlias = `alias and-cli-dev='${pathToCli}'`;
```

Comment:
Just a note for anyone reading this change - these two lines were missed updates in the recent refactor for the Mac/Linux cross-platform issue - most filenames/references were updated but these strings were missed. Not sure how the spacing for the alias got here, but that breaks in bash. Looks like we'll want to write integration tests for the `install` command soon.